### DISCUSSION TITLE
How to transmit on a voice channel?

### DISCUSSION THREAD

**Discussion by ghost** [Rapptz/discord.py#6324] [Q&A]
hi, I need help.  I am currently creating a custom discord client, as you can call discord mod or something else.  But nothing outside of the terms and conditions is somewhat improved. And I'm working with discord.py to create what I say and I got to the point of having to add the function of transmitting a video or its live screen on a voice channel.  But above all, transmit a video that the user chose.  How could I do it?, no matter the difficulty, I'm already strong, Thanks.

**Comment by Zomatree** [comment]
Discord does not have go live or transmit camera documented, so you are on your own in figuring out what to do.

**Reply by ghost** [reply]
ok, thanks.
